src/main/scala/org/fortysevendeg/sparksftp/common/HiveUserData.scala

```diff
@@ -30,7 +28,7 @@ object HiveUserData {
     sparkSession.sql("show tables").show(truncate = false)
   }
 
-  def readUserData(sparkSession: SparkSession): (DataFrame, DataFrame) = {
+  def readUserData(sparkSession: SparkSession): (DataFrame, DataFrame, DataFrame) = {
```

Comment:
I'd create a `case class` to at least have the field names. From the signature perspective, we don't know what is returning this method.

Reply:
Changed in 04a4e304cab1a79eebafe10125719da38e0dda97